src/test/integrasjonstester/kotlin/no/nav/familie/ks/sak/api/BehandlingControllerTest.kt

```diff
@@ -90,6 +90,8 @@ class BehandlingControllerTest : OppslagSpringRunnerTest() {
     @Test
     fun `endreBehandlendeEnhet - skal kaste FunksjonellFeil hvis begrunnelse er tom`() {
         val token = lokalTestToken(behandlerRolle = BehandlerRolle.BESLUTTER)
+        println(token)
```

Comment:
Meningen at `println(token)` skal med?

Reply:
Nice catch, denne var for debugging purposes 😊 
Fjernet nå!